web/src/app/admin/users/users.component.ts

```diff
@@ -88,4 +88,8 @@ export class UsersComponent extends RbacEntityListDirective<RbacUserModel> imple
     return row.username;
   }
 
+  isRowActive = (value) => {
+    return !value?.is_active;
```

Comment:
Are you sure about inverting is_active?

Reply:
Yes, because we are looking for is_active === false, and that's when rows will be grey and all inputs in details form should be disabled